icons/src/fullcolor/render-bitmaps.py

```diff
@@ -236,6 +235,12 @@ def characters(self, chars):
 
 parser = argparse.ArgumentParser(description="Render icons from SVG to PNG")
 
+parser.add_argument(
```

Comment:
I wonder if you would consider using choices instead of a simple string value (like this https://docs.python.org/3/library/argparse.html#choices).
I see the script use mkdir if the DEST directory does not exist, but we could add a bit more error control if permit to render bitmaps only of some supported variants, in this case "default" and "mate".
Pros are the error check in case of mispelling, Cons is that to add support for a new flavour, we should also modify this script, but it doesn't seem too bad from my point of view.

What do you think?

Reply:
Thanks for the suggestion @clobrano :)